tasks.cr

```diff
@@ -3,19 +3,5 @@ require "./src/avram"
 require "./config/*"
 require "./db/migrations/*"
 
-class Db::Reset < LuckyCli::Task
```

Comment:
There is already a separate `Db::Reset` task that was overwriting this one. This was old and I noticed it while updating tasks to the new pattern